packages/@aws-cdk/aws-apigateway/test/test.lambda.ts

```diff
@@ -259,4 +259,46 @@ export = {
 
     test.done();
   },
+
+  'bind works for integration with imported functions'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const restapi = new apigateway.RestApi(stack, 'RestApi');
+    const method = restapi.root.addMethod('ANY');
+    const handler = lambda.Function.fromFunctionArn(stack, 'MyFunc', 'arn:aws:lambda:region:account:function:myfunc');
+    const integration = new apigateway.LambdaIntegration(handler);
+
+    // WHEN
+    const bindResult = integration.bind(method);
+
+    // the deployment token should be defined since the function name
+    // should be a literal string.
+    test.equal(bindResult?.deploymentToken, JSON.stringify({functionName: 'myfunc'}));
+
+    test.done();
+  },
+
+  'bind works for integration with new functions with given name'(test: Test) {
```

Comment:
I believe this test already exists on line 219 in a slightly different form.

Reply:
You're right. let me get rid of it. 

Thanks